server/src/main/java/org/elasticsearch/action/admin/indices/stats/IndicesStatsResponse.java

```diff
@@ -185,13 +186,14 @@ public void writeTo(StreamOutput out) throws IOException {
     }
 
     @Override
-    protected Iterator<ToXContent> customXContentChunks(ToXContent.Params params) {
+    protected Iterator<ToXContent> customXContentChunks(ToXContent.Params outterParams) {
+        var params = new ToXContent.DelegatingMapParams(Map.of(DenseVectorStats.INCLUDE_OFF_HEAP, "true"), outterParams);
```

Comment:
So we want to default to `DenseVectorStats.INCLUDE_OFF_HEAP, "true"`? Reading delegating params it seems that if `outterParams` had `DenseVectorStats.INCLUDE_OFF_HEAP, "false"` it would be overridden.

Reply:
> if outterParams had DenseVectorStats.INCLUDE_OFF_HEAP, "false" it would be overridden
yes, that is true. It seems unlikely that that would be the case, but I can make this conditional so that it does not get overridden.

Reply:
Done in [505f2b4](https://github.com/elastic/elasticsearch/pull/126704/commits/505f2b4b280e81d3685558b32cc153f8088d9d49).